flow-typed/types.js

```diff
@@ -296,35 +296,40 @@ declare type NetworksData = {
 /**
  * Digital assets
  */
-declare type DigitalAssetsListType = 'balance' | 'popular' | 'custom'
+declare type DigitalAssetsFilter = {|
+  +hideZeroBalance: boolean,
+  +myAssetsFirst: boolean,
+  +sortByName: false | 'asc' | 'desc',
+  +sortByBalace: false | 'asc' | 'desc',
+|}
+
+declare type DigitalAssetBalances = {
+  [string]: {
+    balance: Bignumber,
+    lastUpdated: ?Date,
+    isLoading: boolean,
+    isError: boolean,
+  }
+}
 
-declare type DigitalAsset = {
+declare type DigitalAsset = {|
   +address: Address,
   +symbol: string,
   +name: string,
   +decimals: Decimals,
   +isCustom: boolean,
   +isActive: boolean,
-}
-
-declare type DigitalAssetMainDataWithBalance = {
-  +address: Address,
-  +symbol: string,
-  +name: string,
-  +balance: number,
-}
+  // +networkId: number,
```

Comment:
> +networkId: number,

better to avoid specifying of this param for each item. Instead of this we can store assets this way:
(where keys are chain Ids)
```
{
  1: [...],
  3: [...],
  ...
}